packages/now-go/build.sh

```diff
@@ -1,4 +1,16 @@
+#!/bin/bash
+set -euo pipefail
+
+# Start fresh
+rm -rf dist
+
+# Build with `ncc`
 ncc build index.ts -e @vercel/build-utils -e @now/build-utils -o dist
 ncc build install.ts -e @vercel/build-utils -e @now/build-utils -o dist/install
+
+# Move `install.js` to dist
 mv dist/install/index.js dist/install.js
 rm -rf dist/install
+
+# For some reason `ncc` isn't bundling this file
+cp dev-server.go dist
```

Comment:
Should we try bumping ncc to the latest version?

Reply:
I think this line isn't necessary anymore actually. I'll double-check.

Reply:
Removed.